ui/js/service/http.service.ts

```diff
@@ -64,6 +64,15 @@ export class HTTPService {
     );
   }
 
+  authHeader(options?: any): string {
```

Comment:
This is very nitpicking but would it be alright if you could put this method right after the `put` method? I would like to have  our CRUD methods be read first before reading any other additional methods that we may want to add to the http service.

Reply:
👍